### ISSUE TITLE
Current directory for CMake build no longer working in 0.30.1

### ISSUE THREAD

**Issue by CausticYarn** [conan-io/conan#2152]
It appears that the `build_dir` and `source_dir` arguments to `Cmake.configure` used to be absolute, so the below conanfile fragment (taken from the stable bzip2 1.0.6 library on the conan-center site) would work:

```
build(self):
shutil.move("CMakeLists.txt", "%s/CMakeLists.txt" % self.zip_folder_name)
with tools.chdir(self.zip_folder_name):
    os.mkdir("_build")
    with tools.chdir("_build"):
        cmake = CMake(self)
        if self.options.fPIC:
            cmake.definitions["FPIC"] = "ON"
        cmake.configure(build_dir=".", source_dir="..")
        cmake.build(build_dir=".")
```

Now I get a message that the build is unable to find the CMakeLists.txt, as it seems that the `build_dir` and `source_dir` parameters are appended to the build dir (not the current dir, which is ignored).

Is this expected behavior? The conanfile can be rewritten to accommodate the change, but I didn't see this listed in the breaking changes, and it seems like there are existing Conan Center packages that need to be fixed to allow this to work.

**Comment by memsharded** [comment]
Yes, you are right. We are working on fixing this, might release a minor asap. Thanks for reporting.

**Comment by lasote** [comment]
We've released 0.30.2, please try it and tell us if there is any problem.
